services.json

```diff
@@ -40,32 +51,21 @@
 			"title": "solid.authing",
 			"title_color": "#fff",
 			"policyURL": "https://solid.authing.cn",
-			"description": "solid.authing is a public solid server deployed in China",
+			"description": "solid.authing is a public Solid server deployed in China.",
 			"btn_bg": "#43C47A",
 			"btn_color": "#fff"
 		},
 		{
-			"url": "https://solid.blockcasting.io/",
-			"icon": "https://solid.blockcasting.io/blockcasting_logo.png",
-			"icon_bg": "#ffffff",
-			"title": "solid.blockcasting",
+			"url": "https://solid.open.ac.uk/",
+			"icon": "https://www.liberaldictionary.com/wp-content/uploads/2018/12/open-university.png",
+			"icon_bg": "#333748",
+			"title": "solid.open.ac.uk",
 			"title_color": "#fff",
-			"policyURL": "https://solid.blockcasting.io",
-			"description": "solid.blockcasting is a public solid server deployed in Tokyo by Blockcasting Corp, http://blockcasting.io",
+			"policyURL": "http://www.open.ac.uk",
+			"description": "solid.open.ac.uk is a public Solid server based in the UK.",
```

Comment:
s/public Solid server based in the UK./public Solid server at the UK's Open University./

Reply:
inserted details as proposed.